src/sipmanalyze/estimate.py

```diff
@@ -0,0 +1,121 @@
+"""
+
+estimate.py
+
+Functions to help with estimation of low-light parameters.
+
+"""
+
+import sys
+sys.path.insert(0, '../../sipmpdf')
+sys.path.insert(0, '../')
+
+import matplotlib.pyplot as plt
```

Comment:
Consider using the `import sipmanalyze.plotting as plot` for plotting routines. There are containers there to help ensure a uniform plotting style.